usb/usb.go

```diff
@@ -32,7 +32,7 @@ var (
 	init=/init
 rootwait
 `
-	apt      = flag.Bool("apt", false, "apt-get all the things we need")
+	apt      = flag.Bool("apt", true, "apt-get all the things we need")
```

Comment:
we used to have this true. It got to be a problem for travis among other things. I think it ought to stay false.

Reply:
Can you explain the problem more?

Another option is to exit or panic if we don't have the necessary dependencies to compile the kernel.